sdk/identity/Azure.Identity/tests/DefaultAzureCredentialTests.cs

```diff
@@ -536,17 +543,18 @@ public async Task CredentialType(Type availableCredential)
                 ExcludeManagedIdentityCredential = false,
                 ExcludeSharedTokenCacheCredential = false,
                 ExcludeAzureCliCredential = false,
-                ExcludeInteractiveBrowserCredential = false
+                ExcludeInteractiveBrowserCredential = false,
+                Diagnostics =
+                {
+                    IsLoggingContentEnabled = true
```

Comment:
Curious why this is needed. Shouldn't the event be emitted even without logging response content?

Reply:
It may not be, I just assumed it was needed.